### ISSUE TITLE
Add icon and screenshots to F-droid

### ISSUE THREAD

**Issue by inson1** [Yubico/yubioath-flutter#1402]

**Comment by inson1** [comment]
@AdamVe Wouldnt be better to have this in this repostitory?

**Comment by licaon-kter** [comment]
https://gitlab.com/-/snippets/1895688 and in `images/icon.png` is what you want and screenshots and etc

I mean do add description, summary too, and we'll remove them from the metadata https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.yubico.yubioath.yml#L12

these will be shown with the first release that has them

**Comment by AdamVe** [comment]
Hello, thank you all for your involvement! I’ve submitted a PR for adding an app icon (https://github.com/Yubico/yubioath-flutter/pull/1734). Is there any way to check if the icon will be used without having to create a new release of Yubico Authenticator?

@leoheitmannruiz I appreciate your contribution to the documentation—it clearly explains that the app's source repository is the only location where the mentioned image resources can be provided.

**Comment by licaon-kter** [comment]
@AdamVe should be fine

**Comment by AdamVe** [comment]
> ... prefer the [Triple-T structure](https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/#triple-t-structure). That way you won't have to pollute the root of the repository ...

@leoheitmannruiz Agreed and updated.

**Comment by AdamVe** [comment]
I've added the icon to our main branch. We’ll likely need to wait for the next release to see the changes reflected on F-Droid. Let's keep this issue open until we can verify that the icon is displaying correctly.

**Comment by AdamVe** [comment]
Thanks, let me try again. I must have missed that the file was "linked" in an incorrect way.

**Comment by AdamVe** [comment]
Now I hopefully got that correct. The new patch says 120000 for the mode.

https://github.com/Yubico/yubioath-flutter/commit/f0cd11a553aaecb46dfb9159b79c742af01e328e.patch

I misunderstood that last time - I thought that the link would be created by having a text file with the relative path, now I am using a real symbolic link.
